### PR TITLE
PackageInstall: default SyncPeriod to 10 minutes

### PR DESCRIPTION
**Pull request by joe-kimmel-vmw**

### CODE REVIEW

**Review by danielhelfand — APPROVED** [review]
lgtm

Just a note that we should also update documentation on carvel.dev to call out this change.

File: pkg/packageinstall/app.go [hunk 1]
```diff
@@ -7,12 +7,14 @@ import (
...
 	"sort"
 	"strings"
+	"time"
 
 	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/kappctrl/v1alpha1"
 	kcv1alpha1 "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/kappctrl/v1alpha1"
 	pkgingv1alpha1 "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/packaging/v1alpha1"
 	datapkgingv1alpha1 "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apiserver/apis/datapackaging/v1alpha1"
 	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/client/clientset/versioned/scheme"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (cppforlife) [inline pkg/packageinstall/app.go]: nit: use metav1 alias for consistency with other files

File: pkg/packageinstall/app_test.go [hunk 2]
```diff
@@ -14,8 +15,12 @@ import (
 	datapkgingv1alpha1 "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apiserver/apis/datapackaging/v1alpha1"
 	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/packageinstall"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (cppforlife) [inline pkg/packageinstall/app_test.go]: nit: use metav1 alias for consistency with other files

File: pkg/pkgrepository/package_repo_app.go [hunk 3]
```diff
@@ -96,7 +96,7 @@ metadata:
 	}
 
 	if pkgRepository.Spec.SyncPeriod == nil {
-		desiredApp.Spec.SyncPeriod = &metav1.Duration{Duration: time.Minute * 5}
+		desiredApp.Spec.SyncPeriod = &metav1.Duration{Duration: time.Minute * 10}
```
Comment (danielhelfand) [inline pkg/pkgrepository/package_repo_app.go:99]: Are we doing this for repositories as well? Was under impressions only PackageInstalls.

**Pull request merged**